### PR TITLE
Enable windows support for OVTF

### PR DESCRIPTION
**Pull request by ck-intel**

### CODE REVIEW

File: examples/classification_sample/CMakeLists.txt [hunk 1]
```diff
@@ -4,8 +4,8 @@
 # SPDX-License-Identifier: Apache-2.0
 # ******************************************************************************
 
-include_directories(${TensorFlow_INCLUDE_DIR})
-include_directories(${TensorFlow_INCLUDE_DIR}/external/nsync/public)
+# include_directories(${TensorFlow_INCLUDE_DIR})
```
Comment (sspintel) [inline examples/classification_sample/CMakeLists.txt:7]: Can we delete these commented out lines?

File: test/CMakeLists.txt [hunk 2]
```diff
@@ -21,7 +21,7 @@ install(
     PATTERN "*.pb"
     PATTERN "tests_*.txt"
 )
-
+return()
```
Comment (sspintel) [inline test/CMakeLists.txt:24]: Is this required?

File: tools/build_utils.py [hunk 3]
```diff
@@ -101,18 +103,30 @@ def cmake_build(build_dir, src_location, cmake_flags, verbose):
...
     import psutil
     num_cores = str(psutil.cpu_count(logical=True))
-    cmd = ["make", "-j" + num_cores]
-    if verbose:
-        cmd.extend(['VERBOSE=1'])
-    command_executor(cmd, verbose=True)
-    cmd = ["make", "install"]
-    command_executor(cmd, verbose=True)
+    if (platform.system() == 'Windows'):
+        cmd = [
```
Comment (sspintel) [inline tools/build_utils.py:116]: Maybe add a TODO for debug builds later?

File: tools/build_utils.py [hunk 4]
```diff
@@ -190,17 +220,23 @@ def setup_venv(venv_dir):
...
+        command_executor(["python -m pip", "install", "-U", "pip"])
+    else:
+        command_executor(["pip", "install", "-U", "pip"])
     package_list = [
         "pip3",
         "install",
         "psutil",
         "six>=1.12.0",
-        "numpy>=1.16.0,<1.19.0",
+        # "numpy>=1.19.5",
```
Comment (sspintel) [inline tools/build_utils.py:239]: Can you please remove the comment?

File: tools/build_utils.py [hunk 5]
```diff
@@ -276,16 +320,22 @@ def build_tensorflow(tf_version,
...
+        os.environ["CC_OPT_FLAGS"] = "/arch:AVX"
+        command_executor(["python", "configure.py"])
+    else:
+        command_executor("./configure")
+
+    cmd = ["bazel", "build"]
+
+    if (platform.system() != 'Windows'):
+        cmd.extend([
+            "--config=noaws",
```
Comment (sspintel) [inline tools/build_utils.py:334]: --config=opt seems to be missing after the change

File: tools/build_utils.py [hunk 6]
```diff
@@ -721,12 +832,22 @@ def build_openvino(build_dir, openvino_src_dir, cxx_abi, target_arch,
         "-DENABLE_SAMPLES=OFF", "-DENABLE_FUNCTIONAL_TESTS=OFF",
         "-DENABLE_VPU=ON", "-DENABLE_GNA=OFF",
         "-DNGRAPH_ONNX_IMPORT_ENABLE=OFF", "-DNGRAPH_TEST_UTIL_ENABLE=OFF",
-        "-DNGRAPH_COMPONENT_PREFIX=deployment_tools/ngraph/",
         "-DNGRAPH_USE_CXX_ABI=" + cxx_abi, "-DENABLE_CPPLINT=OFF",
-        "-DENABLE_SPEECH_DEMO=FALSE", "-DCMAKE_INSTALL_RPATH=\"$ORIGIN\"",
-        "-DCMAKE_INSTALL_PREFIX=" + install_location
```
Comment (sspintel) [inline tools/build_utils.py:727]: CMAKE_INSTALL_PREFIX must be needed for Windows OV builds also, right?

**Review by sspintel — APPROVED** [review]
Changes look good

File: examples/classification_sample/CMakeLists.txt (hunk 1 above)
Comment (ck-intel) [reply examples/classification_sample/CMakeLists.txt:7]: Done

File: test/CMakeLists.txt (hunk 2 above)
Comment (ck-intel) [reply test/CMakeLists.txt:24]: this condition is required for windows as gtests are not enabled yet,  modified it accordingly

File: tools/build_utils.py (hunk 3 above)
Comment (ck-intel) [reply tools/build_utils.py:116]: done

File: tools/build_utils.py (hunk 4 above)
Comment (ck-intel) [reply tools/build_utils.py:239]: TF on Windows needs a higher version of numpy, handled conditionally

File: tools/build_utils.py (hunk 5 above)
Comment (ck-intel) [reply tools/build_utils.py:334]: done, this flag is needed for windows too

**Review by cavusmustafa — APPROVED** [review]

File: test/CMakeLists.txt (hunk 2 above)
Comment (rrajore) [reply test/CMakeLists.txt]: so can it be under WIN flag, will it have any impact on other OSes

**Comment by rrajore** [comment]
Changes look good!!!

**Pull request merged**